package.json

```diff
@@ -49,19 +49,13 @@
     "lint:other": "yarn prettier --list-different",
     "prettier": "prettier \"**/*.{json,md,scss,yml}\"",
     "release": "lerna publish --no-verify-access && (lerna publish from-package --no-verify-access || true)",
-    "test": "yarn && node bin/testUpdated.js",
-    "test:all": "yarn && yarn dist && lerna run --no-sort --concurrency 1 test"
+    "test": "node bin/testUpdated.js",
+    "test:all": "yarn dist && lerna run --no-sort --concurrency 8 test"
   },
   "workspaces": {
-    "nohoist": [
-      "**/@jest/**/*",
-      "**/@types/!(long)**/*",
-      "**/emotion-normalize",
-      "**/jest-emotion",
-      "**/react"
-    ],
```

Comment:
What was this?

Reply:
This is yarn hoisting (decides where to place npm modules). Its handled different in yarn2 (take a look into the yarn.yml), so this code is not needed anymore (yarn actually warns you that is deprecated)